<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>12000</width>
    <height>8000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    color: #fff;
    background-color: #1890ff;
    border-color: #1890ff;
    border: 1px solid transparent;
    border-radius: 4px;
    line-height: 1.5;
    font-weight: 400;
    white-space: nowrap;
    height: 32px;
}

QLabel {
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Hiragino Sans GB', 'Microsoft YaHei', 'Helvetica Neue', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
    color: #0d1a26;
    font-weight: 500;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1038</width>
      <height>609</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="localIlluminationLabel">
           <property name="text">
            <string>局部光照</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PictureLabel" name="imageLabel">
           <property name="minimumSize">
            <size>
             <width>512</width>
             <height>512</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>512</width>
             <height>512</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="globalIlluminationLabel">
           <property name="text">
            <string>全局光照</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="imageWithGlobalIlluminationLabel">
           <property name="minimumSize">
            <size>
             <width>512</width>
             <height>512</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>512</width>
             <height>512</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="loadObjectButton">
         <property name="text">
          <string>导入模型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="projectionButton">
         <property name="text">
          <string>透视投影</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="renderButton">
         <property name="text">
          <string>全局光照渲染</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureLabel</class>
   <extends>QLabel</extends>
   <header location="global">pictureLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
